test/appium/tests/critical/test_pairing_devices_sync.py

```diff
@@ -148,9 +149,15 @@ def prepare_devices(self):
         dapp = self.home_1.dapp_tab_button.click()
         web_page = dapp.open_url('status.im')
         web_page.add_to_bookmarks()
-        self.home_1.profile_button.double_click()
+
+        self.home_1.just_fyi("Close the ENS banner")
+        [home.home_button.double_click() for home in (self.home_1, self.home_2)]
+        for key in self.drivers:
```

Comment:
What if to replace the lines 155, 156. 157 by [home.ens_banner_close_button.click() for home in (self.home_1, self.home_2)] , will it work?

Reply:
Yes, it works. I just overcomplicated it for some reason :) will push the fix, thank you